unity-renderer/Assets/Scripts/MainScripts/DCL/WorldRuntime/SceneController.cs

```diff
@@ -5,11 +5,14 @@
 using DCL.Configuration;
 using System;
 using System.Collections;
-using System.Collections.Generic;
+using System.Collections.Concurrent;
 using System.Linq;
+using System.Threading;
+using Cysharp.Threading.Tasks;
 using DCL.Components;
-using Newtonsoft.Json;
+using JetBrains.Annotations;
```

Comment:
```suggestion
```
I think this assembly is rider specific.

Reply:
Yes this was a mistake